modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -557,6 +557,25 @@ public void testImportObjectDefinition() throws Exception {
 			_objectDefinitionImportFieldNames, jsonNode.get(0));
 	}
 
+	@Test
+	public void testImportSiteScopeObjectEntry() throws Exception {
+		_objectDefinition1 = _publishObjectDefinition(
+			TestPropsValues.getCompanyId(), "TestObject",
+			ObjectDefinitionConstants.SCOPE_SITE, TestPropsValues.getUser());
+
+		// default group
+
+		_testImportSiteScopeObjectEntry(
+			TestPropsValues.getGroupId(), _OBJECT_ENTRY_ERC_1);
+
+		Group group = GroupTestUtil.addGroup();
```

Comment:
Is this deleted after the test runs?

I think so, but... can you double check?

Reply:
Create the group in the setup and annotate it with @DeleteAfterTestRun.